internal/controller/controllers/local_cluster_import_controller.go

```diff
@@ -134,7 +138,7 @@ func (r *LocalClusterImportReconciler) Reconcile(ctx context.Context, req ctrl.R
 		}
 		return ctrl.Result{}, errors.Wrap(err, "error while attempting to determine presence of managed cluster")
 	}
-	if hasManagedCluster {
+	if hasManagedCluster && importLocalClusterEnabled {
```

Comment:
After some input from @berenss it seems like it's possible for some users to have created these resources (effectively importing the local cluster) before this controller existed.

Making this change would delete their resources which we should avoid.

Instead of this let's just not reconcile at all unless we're enabled. Then users can recover the situation based on their individual situation.

Reply:
Updated to force early exit of reconcile with no changes/deletions of CRs.